index.html

```diff
@@ -353,8 +353,10 @@ <h2>Securing JSON-LD Verifiable Credentials with SD-JWT</h2>
           to [[VC-DATA-MODEL-2.0]].
         </p>
         <p>
-          A [=conforming SD-JWT issuer implementation=] MUST use [[SD-JWT]] to secure this media type.
-	  The unsecured verifiable credential is the unencoded SD-JWT payload.
+A [=conforming SD-JWT issuer implementation=] MUST use [[[SD-JWT]]] to secure
+this media type. The unsecured verifiable credential is the JSON input claim
+set. The Issuer then converts the input claim set (i.e., the unsecured
+verifiable credential) into an SD-JWT payload according to [[SD-JWT]].
```

Comment:
```suggestion
this media type. The unsecured [=verifiable credential=] is the input JSON
claim set. The Issuer then converts the input JSON claim set (i.e., the
unsecured [=verifiable credential=]) into an SD-JWT payload according to
[[[SD-JWT]]] [[SD-JWT]].
```

Reply:
Why do we want both triple brackets and double brackets?